ImperatorToCK3/Data_Files/blankMod/output/common/religion/religions/00_kushitism.txt

```diff
@@ -1,4 +1,4 @@
-﻿
```

Comment:
I think you reverted to UTF8. The file should be in UTF8-BOM.

Reply:
It didn't give me any problem in game. Do I need to re-encode that?

Reply:
The game logs about it in error.log. If you use Notepad++, you just need to select "Convert to UTF8-BOM" or sth like that.
![image](https://user-images.githubusercontent.com/29546927/111212931-88d0a900-85d0-11eb-9201-79ea93cfd000.png)

Reply:
Done, for both